src/shared/Core/Interop/Windows/Native/Shell32.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace GitCredentialManager.Interop.Windows.Native
+{
+    public static class Shell32
+    {
+        private const string LibraryName = "shell32.dll";
+
+        /// <summary>
+        /// Parses a Unicode command line string and returns an array of pointers
+        /// to the command line arguments, along with a count of such arguments,
+        /// in a way that is similar to the standard C run-time argv and argc values.
+        /// </summary>
+        /// <param name="lpCmdLine">
+        /// Pointer to a null-terminated Unicode string that contains the full command line.
+        /// If this parameter is an empty string the function returns the path to the current executable file.
+        /// </param>
+        /// <param name="pNumArgs">
+        /// Pointer to an int that receives the number of array elements returned, similar to argc.
+        /// </param>
+        /// <returns>A pointer to an array of LPWSTR values, similar to argv.</returns>
+        [DllImport("Shell32.dll", CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Unicode, SetLastError = true)]
+        public static extern IntPtr CommandLineToArgvW(IntPtr lpCmdLine, out int pNumArgs);
```

Comment:
Do we need to import `LocalFree()`, too? That's [what I call](https://github.com/git-for-windows/MINGW-packages/blob/764f4bab64a06db7883188e12fe1bb458baab7ad/mingw-w64-git/git-wrapper.c#L361) in [the Git wrapper](https://github.com/git-for-windows/git/wiki/Git-wrapper) after [splitting the command line](https://github.com/git-for-windows/MINGW-packages/blob/764f4bab64a06db7883188e12fe1bb458baab7ad/mingw-w64-git/git-wrapper.c#L277).

Reply:
Added!